circuits/results/helpers.go

```diff
@@ -0,0 +1,77 @@
+package results
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/consensys/gnark/frontend"
+	"github.com/vocdoni/vocdoni-z-sandbox/circuits/merkleproof"
+	"github.com/vocdoni/vocdoni-z-sandbox/crypto/elgamal"
+	"github.com/vocdoni/vocdoni-z-sandbox/state"
+	"github.com/vocdoni/vocdoni-z-sandbox/types"
+)
+
+func GenerateWitness(
+	o *state.State,
+	results [types.FieldsPerBallot]*big.Int,
+	addResults [types.FieldsPerBallot]*big.Int,
+	subResults [types.FieldsPerBallot]*big.Int,
+	addCiphertexts [types.FieldsPerBallot]elgamal.Ciphertext,
+	subCiphertexts [types.FieldsPerBallot]elgamal.Ciphertext,
+	decryptionAddProofs [types.FieldsPerBallot]*elgamal.DecryptionProof,
```

Comment:
I think names could be a bit more consistent. Let me do some proposals.

addAccumulator
subAccumulator
addAccumulatorEncrypted
subAccumulatorEncrypted
addAccumulatorMkProof
subAccumulatorMkProof

In general, anything that is a "Merkle Proof", should be named MkProof or MerkleProof, because we use proof to refer zkSNARK proofs, usually.

Reply:
And same with `Proof` prefix.

Reply:
We must always choose the best option, not just the existing one. So if there is something which needs to be changed, we do it (if not on the same PR, just let's open an issue)